class-debug-bar-cron.php

```diff
@@ -380,6 +382,21 @@ private function display_schedules() {
 				</table>';
 		}
 
+		/**
+		 * Sorting method for cron scheldules. Order by schedules interval.
+		 *
+		 * @param array $a First element of comparison pair.
+		 * @param array $b Second element of comparison pair.
+		 *
+		 * @return int Return 1 if $a argument 'interval' greater then $b argument 'interval', 0 if both intervals equivalent and -1 otherwise.
+		 */
+		function schedules_sorting( $a, $b ){
+			if ( (int) $a['interval'] === (int) $b['interval'] ){
+				return 0;
+			} else {
+				return (int) $a['interval'] > (int) $b['interval'] ? 1 : -1;
```

Comment:
Ternary comparisons should be bracketed to ensure that there can be no confusion as to what is part of the ternary and what should be returned.

So this should be changed to:
```php
return ( ( (int) $a['interval'] > (int) $b['interval'] ) ? 1 : -1 );
```